src/components/filters/FilterGroup/index.jsx

```diff
@@ -39,8 +39,17 @@ class FilterGroup extends React.Component {
     const initialExpandedStatusControl = true;
     const initialExpandedStatus = props.filterConfig.tabs
       .map(t => t.fields.map(() => (initialExpandedStatusControl)));
-    const initialFilterStatus = props.filterConfig.tabs
+    let initialFilterStatus = props.filterConfig.tabs
       .map(t => t.fields.map(() => ({})));
+
+    if (props.filterStatusFromParent && Object.keys(props.filterStatusFromParent).length > 0) {
```

Comment:
any reason for not just checking `filterStatusFromParent` length here? it is also an array

Reply:
the default value of the prop is undefined, not the empty array. so i check if it is defined. it's an optional parameter.

Reply:
oh good catch, hmmm then I think you should use the same logic for `filterStatusFromParent` in `FilterList` as well (I know I have suggested to change that one to checking the length too)

I think it is okay to make an array props defaults to `[]` even if it is optional, but it is your call

Reply:
ahh i see what ur saying. yes, i should check the length instead of Object.keys

Reply:
looks good! thank you